DevoxxClientMobile/src/main/java/com/devoxx/views/ConfSelectorPresenter.java

```diff
@@ -116,41 +122,52 @@ public void initialize() {
             });
             e.consume();
         });
-
         filterPane.getChildren().add(filter);
-        
     }
 
     private MenuPopupView getMenuPopupView(Button filter) {
         Menu menu = new Menu();
         MenuItem devoxx = new MenuItem(Conference.Type.DEVOXX.toString());
         MenuItem voxxed = new MenuItem(Conference.Type.VOXXED.toString());
+        MenuItem futureEvents = new MenuItem(bundle.getString("OTN.CONFERENCE_SELECTOR.HEADER.DEVOXX_VOXXED"));
         MenuItem pastEvents = new MenuItem(bundle.getString("OTN.CONFERENCE_SELECTOR.HEADER.PAST_EVENTS"));
-        menu.getItems().addAll(devoxx, voxxed, pastEvents);
+        menu.getItems().addAll(devoxx, voxxed, futureEvents, pastEvents);
         
         devoxx.setOnAction(e -> {
-            selector.setComparator(futureConferenceComparator);
-            selector.setItems(service.retrieveConferences(Conference.Type.DEVOXX));
-            header.setText(Conference.Type.DEVOXX.getDisplayName());
+            updateFutureEvent(Conference.Type.DEVOXX.name());
             STATUS_BAR.pseudoClassStateChanged(PSEUDO_CLASS_STATUS_VOXXED, false);
         });
         voxxed.setOnAction(e -> {
-            selector.setComparator(futureConferenceComparator);
-            selector.setItems(service.retrieveConferences(Conference.Type.VOXXED));
-            header.setText(Conference.Type.VOXXED.getDisplayName());
+            updateFutureEvent(Conference.Type.VOXXED.name());
             STATUS_BAR.pseudoClassStateChanged(PSEUDO_CLASS_STATUS_VOXXED, true);
         });
+        futureEvents.setOnAction(e -> {
+            updateFutureEvent(null);
+            STATUS_BAR.pseudoClassStateChanged(PSEUDO_CLASS_STATUS_VOXXED, false);
+        });
         pastEvents.setOnAction(e -> {
             selector.setComparator(pastConferenceComparator);
             selector.setItems(service.retrievePastConferences());
             header.setText(bundle.getString("OTN.CONFERENCE_SELECTOR.HEADER.PAST_EVENTS"));
             STATUS_BAR.pseudoClassStateChanged(PSEUDO_CLASS_STATUS_VOXXED, false);
         });
-        
 
         final MenuPopupView menuPopupView = new MenuPopupView(filter, menu);
         menuPopupView.getStyleClass().add("conf-selector");
         menuPopupView.setSide(RIGHT);
         return menuPopupView;
     }
+
+    private void updateFutureEvent(String eventName) {
```

Comment:
Why don't you pass in the enum type itself instead of its string representation?

Reply:
I need to pass null to identify when to show both Devoxx and Voxxed event type.

Reply:
I don't understand, Conference.Type can be null too? Now you pass a string that is built from enum.name() and inside the function, you convert the string back into the corresponding enum value.

Reply:
Fixed and pushed as a part of https://github.com/devoxx/MyDevoxxGluon/pull/168